packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -834,11 +760,7 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
                         "Error: You have Zowe USS favorites that refer to a non-existent CLI profile named: {0}",
                         profileName
                     ) +
-                    localize(
-                        "intializeUSSFavorites.error.profile2",
-                        ". To resolve this, you can remove {0}",
-                        profileName
-                    ) +
+                    localize("intializeUSSFavorites.error.profile2", ". To resolve this, you can remove {0}", profileName) +
```

Comment:
Should we rename `intializeUSSFavorites` -> `initializeUSSFavorites` to fix the typo? We could create a separate issue for this as well, since I know it's outside the scope of this PR 🙂

Reply:
Fixed in new commit 👍